docs/api_reference/src/quantities.tex

```diff
@@ -1,266 +1,6718 @@
+\begin{itemdecl}
+template<@\libconcept{PointOrigin}@ PO1, @\libconcept{PointOrigin}@ PO2>
+friend consteval bool operator==(PO1 po1, PO2 po2);
+\end{itemdecl}
+
+\begin{itemdescr}
+\pnum
+\effects
+Equivalent to:
+\begin{codeblock}
+if constexpr (@\exposidnc{is-derived-from-specialization-of}@<PO1, absolute_point_origin>() &&
+              @\exposidnc{is-derived-from-specialization-of}@<PO2, absolute_point_origin>())
+  return std::is_same_v<PO1, PO2> ||
+         (@\exposidnc{is-specialization-of}@<PO1, zeroth_point_origin>() &&
+          @\exposidnc{is-specialization-of}@<PO2, zeroth_point_origin>() &&
+          interconvertible(po1.@\exposidnc{quantity-spec}@, po2.@\exposidnc{quantity-spec}@));
+else if constexpr (@\exposidnc{is-derived-from-specialization-of}@<PO1, relative_point_origin>() &&
+                   @\exposidnc{is-derived-from-specialization-of}@<PO2, relative_point_origin>())
+  return PO1::@\exposidnc{quantity-point}@ == PO2::@\exposidnc{quantity-point}@;
+else if constexpr (@\exposidnc{is-derived-from-specialization-of}@<PO1, relative_point_origin>())
+  return @\exposidnc{same-absolute-point-origins}@(po1, po2) &&
+         is_eq_zero(PO1::@\exposidnc{quantity-point}@.quantity_from_zero());
+else if constexpr (@\exposidnc{is-derived-from-specialization-of}@<PO2, relative_point_origin>())
+  return @\exposidnc{same-absolute-point-origins}@(po1, po2) &&
+         is_eq_zero(PO2::@\exposidnc{quantity-point}@.quantity_from_zero());
+\end{codeblock}
+\end{itemdescr}
+
+\begin{codeblock}
+namespace mp_units {
+
+template<@\libconcept{QuantitySpec}@ auto QS>
+struct @\libglobal{absolute_point_origin}@ : @\exposidnc{point-origin-interface}@ {
+  static constexpr @\libconcept{QuantitySpec}@ auto @\exposidnc{quantity-spec}@ = QS;  // \expos
+};
+
+}
+\end{codeblock}
+
+\pnum
+An \defnadj{absolute}{origin} is an origin
+chosen by convention and not defined in terms of another origin.
+A specialization of \tcode{absolute_point_origin} is used as a base type when defining an absolute origin.
+\tcode{QS} is the quantity the origin represents.
+
+\begin{codeblock}
+namespace mp_units {
+
+template<@\libconcept{QuantityPoint}@ auto QP>
+struct @\libglobal{relative_point_origin}@ : @\exposidnc{point-origin-interface}@ {
+  static constexpr @\libconcept{QuantityPoint}@ auto @\exposidnc{quantity-point}@ = QP;       // \expos
+  static constexpr @\libconcept{QuantitySpec}@ auto @\exposidnc{quantity-spec}@ = @\seebelownc@;  // \expos
+  static constexpr @\libconcept{PointOrigin}@ auto @\exposidnc{absolute-point-origin}@ =      // \expos
+    QP.absolute_point_origin;
+};
+
+}
+\end{codeblock}
+
+\pnum
+A \defnadj{relative}{origin} is an origin
+of a subspace\irefiev{102-03-03}.
+A specialization of \tcode{relative_point_origin} is used as a base type when defining a relative origin \placeholder{O}.
+\placeholder{O} is offset from \tcode{QP.absolute_point_origin} by \tcode{QP.quantity_from_zero()}.
```

Comment:
Should `QP.quantity_from_zero()` actually be `QP.quantity_from(QP.absolute_point_origin)`?